lib/widget/widget.h

```diff
@@ -233,75 +233,75 @@ void widgRemoveOverlayScreen(W_SCREEN* psScreen);
 bool isMouseOverScreenOverlayChild(int mx, int my); // global mouse coordinates - i.e. those returned from mouseX()/mouseY()
 
 /** Add a form to the widget screen */
-WZ_DECL_NONNULL(1, 2) W_FORM *widgAddForm(W_SCREEN *psScreen, const W_FORMINIT *psInit);
+WZ_DECL_NONNULL(2) W_FORM *widgAddForm(std::shared_ptr<W_SCREEN> const &psScreen, const W_FORMINIT *psInit);
```

Comment:
The codebase is lacking consistency on this point, but I'd like to try to stick with whatever pattern of `type const *` or `const type *` is used in a particular file. For this file, I see a lot more use of `const`-first, so these should probably be `const std::shared_ptr<W_SCREEN> &`?

(At some point we should really run clang-format or similar on the codebase, but I dread how huge that commit will be...)

Reply:
I'm all for having some set of formatting/naming standards. I will change the const order.